modules/codegen-plugin/src/sbt-test/codegen-plugin/defaults/src/main/scala/Main.scala

```diff
@@ -24,4 +27,22 @@ object Main extends App {
       println("failed")
       sys.exit(1)
   }
+
+  def buildSchemaIndex() =
+    ModelLoader
+      .load(
+        specs = Set.empty,
+        dependencies = List(
+          "com.disneystreaming.smithy4s::smithy4s-dynamic:0.15.1",
+          "com.disneystreaming.smithy4s::smithy4s-codegen:0.15.1"
+        ),
```

Comment:
are these necessary? I don't think they bring in a lot of smithy

Reply:
ahh it's my bad, it's a forgotton attempt